es-vue-base/src/lib-components/EsPopover.vue

```diff
@@ -111,11 +121,90 @@ export default {
 <style lang="scss">
 @import '~@energysage/es-bs-base/scss/includes';
 
-.popover-content-wrapper {
-    a {
-        color: #e0eff5;
-        font-weight: $font-weight-bold;
+.es-popover-light {
+    &.popover {
+        background-color: $white;
+        border: 1px solid $primary;
+    }
+    // styling for all arrow backgrounds
+    &.bs-popover-bottom, &.bs-popover-auto[x-placement^=bottom] {
+        > .arrow {
+            &::before {
+                border-bottom-color: $primary;
+            }
+            &::after {
+                top: 1px;
+                border-bottom-color: $white;
+            }
+        }
+    }
+    &.bs-popover-top, &.bs-popover-auto[x-placement^=top] {
+        > .arrow {
+            &::before {
+                border-top-color: $primary;
+            }
+            &::after {
+                bottom: 1px;
+                border-top-color: $white;
+            }
+        }
+    }
+    &.bs-popover-right, &.bs-popover-auto[x-placement^=right] {
+        > .arrow {
+            &::before {
+                border-right-color: $primary;
+            }
+            &::after {
+                left: 1px;
+                border-right-color: $white;
+            }
+        }
+    }
+    &.bs-popover-left, &.bs-popover-auto[x-placement^=left] {
+        > .arrow {
+            &::before {
+                border-left-color: $primary;
+            }
+            &::after {
+                right: 1px;
+                border-left-color: $white;
+            }
+        }
+    }
+    .popover-header {
+        color: $black;
+        background-color: $white;
+        border-bottom-color: $white;
+        &::before {
+            border-bottom-color: $white;
+        }
+        .btn, .btn:hover, .btn:active {
+            color: $black;
+        }
+    }
+    .popover-body {
+        color: $black;
+        .btn, .btn:hover, .btn:active {
+            color: $black;
+        }
     }
 }
 
+.es-popover-dark {
+    // styling for this color scheme will need to be added as default values for popovers are updated
+    .popover-header {
+        .btn, .btn:hover, .btn:active {
+            color: $white;
+        }
+    }
+    .popover-body {
+        .btn, .btn:hover, .btn:active {
+            color: $white;
+        }
+        a {
+        color: #99CBDF;
+        font-weight: $font-weight-bold;
```

Comment:
@RyanMulready I asked design about link styling for both variants and they said that only the `dark` variant has custom styling. The color in ESDS was actually wrong, so I updated it here. I also moved it into the `dark` class definition.